app/pb_kits/playbook/pb_home_address_street/docs/_home_address_street_dark.html.erb

```diff
@@ -1,10 +1,11 @@
 <%= pb_rails("home_address_street", props: {
   address: "70 Prospect Ave",
   address2: "Apt M18",
```

Comment:
Looks like this got merged last week as wasn't picked up. Can you rename this? Address2 is not descriptive enough. Let's go for address_cont or address_continued

Reply:
We can definitely rename it. I went with address and address2 as this is how the addresses exist in Nitro database

Reply:
I like address continued, I see cont apply to "contains" as well in Nitro.  Changing that now.